// @cds/core/tokens/tokens.js
// This token format is currently experimental and may change in the future
export const globalBase = 'var(--cds-global-base)';
export const globalScaleSpace = 'var(--cds-global-scale-space)';
export const globalScaleLayoutSpace = 'var(--cds-global-scale-layout-space)';
export const globalScaleTypography = 'var(--cds-global-scale-typography)';
export const globalLayoutGridCols = 'var(--cds-global-layout-grid-cols)';
export const globalLayoutWidthXs = 'var(--cds-global-layout-width-xs)';
export const globalLayoutWidthSm = 'var(--cds-global-layout-width-sm)';
export const globalLayoutWidthMd = 'var(--cds-global-layout-width-md)';
export const globalLayoutWidthLg = 'var(--cds-global-layout-width-lg)';
export const globalLayoutWidthXl = 'var(--cds-global-layout-width-xl)';
export const globalLayoutSpaceXxxs = 'var(--cds-global-layout-space-xxxs)';
export const globalLayoutSpaceXxs = 'var(--cds-global-layout-space-xxs)';
export const globalLayoutSpaceXs = 'var(--cds-global-layout-space-xs)';
export const globalLayoutSpaceSm = 'var(--cds-global-layout-space-sm)';
export const globalLayoutSpaceMd = 'var(--cds-global-layout-space-md)';
export const globalLayoutSpaceLg = 'var(--cds-global-layout-space-lg)';
export const globalLayoutSpaceXl = 'var(--cds-global-layout-space-xl)';
export const globalLayoutSpaceXxl = 'var(--cds-global-layout-space-xxl)';
export const globalLayoutSpaceXxxl = 'var(--cds-global-layout-space-xxxl)';
export const globalSpace0 = 'var(--cds-global-space-0)';
export const globalSpace1 = 'var(--cds-global-space-1)';
export const globalSpace2 = 'var(--cds-global-space-2)';
export const globalSpace3 = 'var(--cds-global-space-3)';
export const globalSpace4 = 'var(--cds-global-space-4)';
export const globalSpace5 = 'var(--cds-global-space-5)';
export const globalSpace6 = 'var(--cds-global-space-6)';
export const globalSpace7 = 'var(--cds-global-space-7)';
export const globalSpace8 = 'var(--cds-global-space-8)';
export const globalSpace9 = 'var(--cds-global-space-9)';
export const globalSpace10 = 'var(--cds-global-space-10)';
export const globalSpace11 = 'var(--cds-global-space-11)';
export const globalSpace12 = 'var(--cds-global-space-12)';
export const globalSpace13 = 'var(--cds-global-space-13)';
export const globalSpace14 = 'var(--cds-global-space-14)';
export const globalSpace15 = 'var(--cds-global-space-15)';
export const globalColorBlack = 'var(--cds-global-color-black)';
export const globalColorWhite = 'var(--cds-global-color-white)';
export const globalColorGreen50 = 'var(--cds-global-color-green-50)';
export const globalColorGreen100 = 'var(--cds-global-color-green-100)';
export const globalColorGreen200 = 'var(--cds-global-color-green-200)';
export const globalColorGreen300 = 'var(--cds-global-color-green-300)';
export const globalColorGreen400 = 'var(--cds-global-color-green-400)';
export const globalColorGreen500 = 'var(--cds-global-color-green-500)';
export const globalColorGreen600 = 'var(--cds-global-color-green-600)';
export const globalColorGreen700 = 'var(--cds-global-color-green-700)';
export const globalColorGreen800 = 'var(--cds-global-color-green-800)';
export const globalColorGreen900 = 'var(--cds-global-color-green-900)';
export const globalColorGreen1000 = 'var(--cds-global-color-green-1000)';
export const globalColorBlue50 = 'var(--cds-global-color-blue-50)';
export const globalColorBlue75 = 'var(--cds-global-color-blue-75)';
export const globalColorBlue100 = 'var(--cds-global-color-blue-100)';
export const globalColorBlue200 = 'var(--cds-global-color-blue-200)';
export const globalColorBlue300 = 'var(--cds-global-color-blue-300)';
export const globalColorBlue400 = 'var(--cds-global-color-blue-400)';
export const globalColorBlue500 = 'var(--cds-global-color-blue-500)';
export const globalColorBlue600 = 'var(--cds-global-color-blue-600)';
export const globalColorBlue700 = 'var(--cds-global-color-blue-700)';
export const globalColorBlue800 = 'var(--cds-global-color-blue-800)';
export const globalColorBlue900 = 'var(--cds-global-color-blue-900)';
export const globalColorBlue1000 = 'var(--cds-global-color-blue-1000)';
export const globalColorViolet50 = 'var(--cds-global-color-violet-50)';
export const globalColorViolet100 = 'var(--cds-global-color-violet-100)';
export const globalColorViolet200 = 'var(--cds-global-color-violet-200)';
export const globalColorViolet300 = 'var(--cds-global-color-violet-300)';
export const globalColorViolet400 = 'var(--cds-global-color-violet-400)';
export const globalColorViolet500 = 'var(--cds-global-color-violet-500)';
export const globalColorViolet600 = 'var(--cds-global-color-violet-600)';
export const globalColorViolet700 = 'var(--cds-global-color-violet-700)';
export const globalColorViolet800 = 'var(--cds-global-color-violet-800)';
export const globalColorViolet900 = 'var(--cds-global-color-violet-900)';
export const globalColorViolet1000 = 'var(--cds-global-color-violet-1000)';
export const globalColorRed50 = 'var(--cds-global-color-red-50)';
export const globalColorRed100 = 'var(--cds-global-color-red-100)';
export const globalColorRed200 = 'var(--cds-global-color-red-200)';
export const globalColorRed300 = 'var(--cds-global-color-red-300)';
export const globalColorRed400 = 'var(--cds-global-color-red-400)';
export const globalColorRed500 = 'var(--cds-global-color-red-500)';
export const globalColorRed600 = 'var(--cds-global-color-red-600)';
export const globalColorRed700 = 'var(--cds-global-color-red-700)';
export const globalColorRed800 = 'var(--cds-global-color-red-800)';
export const globalColorRed900 = 'var(--cds-global-color-red-900)';
export const globalColorRed1000 = 'var(--cds-global-color-red-1000)';
export const globalColorOchre50 = 'var(--cds-global-color-ochre-50)';
export const globalColorOchre100 = 'var(--cds-global-color-ochre-100)';
export const globalColorOchre200 = 'var(--cds-global-color-ochre-200)';
export const globalColorOchre300 = 'var(--cds-global-color-ochre-300)';
export const globalColorOchre400 = 'var(--cds-global-color-ochre-400)';
export const globalColorOchre500 = 'var(--cds-global-color-ochre-500)';
export const globalColorOchre600 = 'var(--cds-global-color-ochre-600)';
export const globalColorOchre700 = 'var(--cds-global-color-ochre-700)';
export const globalColorOchre800 = 'var(--cds-global-color-ochre-800)';
export const globalColorOchre900 = 'var(--cds-global-color-ochre-900)';
export const globalColorOchre1000 = 'var(--cds-global-color-ochre-1000)';
export const globalColorLavender50 = 'var(--cds-global-color-lavender-50)';
export const globalColorLavender100 = 'var(--cds-global-color-lavender-100)';
export const globalColorLavender200 = 'var(--cds-global-color-lavender-200)';
export const globalColorLavender300 = 'var(--cds-global-color-lavender-300)';
export const globalColorLavender400 = 'var(--cds-global-color-lavender-400)';
export const globalColorLavender500 = 'var(--cds-global-color-lavender-500)';
export const globalColorLavender600 = 'var(--cds-global-color-lavender-600)';
export const globalColorLavender700 = 'var(--cds-global-color-lavender-700)';
export const globalColorLavender800 = 'var(--cds-global-color-lavender-800)';
export const globalColorLavender900 = 'var(--cds-global-color-lavender-900)';
export const globalColorLavender1000 = 'var(--cds-global-color-lavender-1000)';
export const globalColorAzure50 = 'var(--cds-global-color-azure-50)';
export const globalColorAzure100 = 'var(--cds-global-color-azure-100)';
export const globalColorAzure200 = 'var(--cds-global-color-azure-200)';
export const globalColorAzure300 = 'var(--cds-global-color-azure-300)';
export const globalColorAzure400 = 'var(--cds-global-color-azure-400)';
export const globalColorAzure500 = 'var(--cds-global-color-azure-500)';
export const globalColorAzure600 = 'var(--cds-global-color-azure-600)';
export const globalColorAzure700 = 'var(--cds-global-color-azure-700)';
export const globalColorAzure800 = 'var(--cds-global-color-azure-800)';
export const globalColorAzure900 = 'var(--cds-global-color-azure-900)';
export const globalColorAzure1000 = 'var(--cds-global-color-azure-1000)';
export const globalColorAqua50 = 'var(--cds-global-color-aqua-50)';
export const globalColorAqua100 = 'var(--cds-global-color-aqua-100)';
export const globalColorAqua200 = 'var(--cds-global-color-aqua-200)';
export const globalColorAqua300 = 'var(--cds-global-color-aqua-300)';
export const globalColorAqua400 = 'var(--cds-global-color-aqua-400)';
export const globalColorAqua500 = 'var(--cds-global-color-aqua-500)';
export const globalColorAqua600 = 'var(--cds-global-color-aqua-600)';
export const globalColorAqua700 = 'var(--cds-global-color-aqua-700)';
export const globalColorAqua800 = 'var(--cds-global-color-aqua-800)';
export const globalColorAqua900 = 'var(--cds-global-color-aqua-900)';
export const globalColorAqua1000 = 'var(--cds-global-color-aqua-1000)';
export const globalColorJade50 = 'var(--cds-global-color-jade-50)';
export const globalColorJade100 = 'var(--cds-global-color-jade-100)';
export const globalColorJade200 = 'var(--cds-global-color-jade-200)';
export const globalColorJade300 = 'var(--cds-global-color-jade-300)';
export const globalColorJade400 = 'var(--cds-global-color-jade-400)';
export const globalColorJade500 = 'var(--cds-global-color-jade-500)';
export const globalColorJade600 = 'var(--cds-global-color-jade-600)';
export const globalColorJade700 = 'var(--cds-global-color-jade-700)';
export const globalColorJade800 = 'var(--cds-global-color-jade-800)';
export const globalColorJade900 = 'var(--cds-global-color-jade-900)';
export const globalColorJade1000 = 'var(--cds-global-color-jade-1000)';
export const globalColorYellow50 = 'var(--cds-global-color-yellow-50)';
export const globalColorYellow100 = 'var(--cds-global-color-yellow-100)';
export const globalColorYellow200 = 'var(--cds-global-color-yellow-200)';
export const globalColorYellow300 = 'var(--cds-global-color-yellow-300)';
export const globalColorYellow400 = 'var(--cds-global-color-yellow-400)';
export const globalColorYellow500 = 'var(--cds-global-color-yellow-500)';
export const globalColorYellow600 = 'var(--cds-global-color-yellow-600)';
export const globalColorYellow700 = 'var(--cds-global-color-yellow-700)';
export const globalColorYellow800 = 'var(--cds-global-color-yellow-800)';
export const globalColorYellow900 = 'var(--cds-global-color-yellow-900)';
export const globalColorYellow1000 = 'var(--cds-global-color-yellow-1000)';
export const globalColorLime50 = 'var(--cds-global-color-lime-50)';
export const globalColorLime100 = 'var(--cds-global-color-lime-100)';
export const globalColorLime200 = 'var(--cds-global-color-lime-200)';
export const globalColorLime300 = 'var(--cds-global-color-lime-300)';
export const globalColorLime400 = 'var(--cds-global-color-lime-400)';
export const globalColorLime500 = 'var(--cds-global-color-lime-500)';
export const globalColorLime600 = 'var(--cds-global-color-lime-600)';
export const globalColorLime700 = 'var(--cds-global-color-lime-700)';
export const globalColorLime800 = 'var(--cds-global-color-lime-800)';
export const globalColorLime900 = 'var(--cds-global-color-lime-900)';
export const globalColorLime1000 = 'var(--cds-global-color-lime-1000)';
export const globalColorTangerine50 = 'var(--cds-global-color-tangerine-50)';
export const globalColorTangerine100 = 'var(--cds-global-color-tangerine-100)';
export const globalColorTangerine200 = 'var(--cds-global-color-tangerine-200)';
export const globalColorTangerine300 = 'var(--cds-global-color-tangerine-300)';
export const globalColorTangerine400 = 'var(--cds-global-color-tangerine-400)';
export const globalColorTangerine500 = 'var(--cds-global-color-tangerine-500)';
export const globalColorTangerine600 = 'var(--cds-global-color-tangerine-600)';
export const globalColorTangerine700 = 'var(--cds-global-color-tangerine-700)';
export const globalColorTangerine800 = 'var(--cds-global-color-tangerine-800)';
export const globalColorTangerine900 = 'var(--cds-global-color-tangerine-900)';
export const globalColorTangerine1000 = 'var(--cds-global-color-tangerine-1000)';
export const globalColorMagenta50 = 'var(--cds-global-color-magenta-50)';
export const globalColorMagenta100 = 'var(--cds-global-color-magenta-100)';
export const globalColorMagenta200 = 'var(--cds-global-color-magenta-200)';
export const globalColorMagenta300 = 'var(--cds-global-color-magenta-300)';
export const globalColorMagenta400 = 'var(--cds-global-color-magenta-400)';
export const globalColorMagenta500 = 'var(--cds-global-color-magenta-500)';
export const globalColorMagenta600 = 'var(--cds-global-color-magenta-600)';
export const globalColorMagenta700 = 'var(--cds-global-color-magenta-700)';
export const globalColorMagenta800 = 'var(--cds-global-color-magenta-800)';
export const globalColorMagenta900 = 'var(--cds-global-color-magenta-900)';
export const globalColorMagenta1000 = 'var(--cds-global-color-magenta-1000)';
export const globalColorPink50 = 'var(--cds-global-color-pink-50)';
export const globalColorPink100 = 'var(--cds-global-color-pink-100)';
export const globalColorPink200 = 'var(--cds-global-color-pink-200)';
export const globalColorPink300 = 'var(--cds-global-color-pink-300)';
export const globalColorPink400 = 'var(--cds-global-color-pink-400)';
export const globalColorPink500 = 'var(--cds-global-color-pink-500)';
export const globalColorPink600 = 'var(--cds-global-color-pink-600)';
export const globalColorPink700 = 'var(--cds-global-color-pink-700)';
export const globalColorPink800 = 'var(--cds-global-color-pink-800)';
export const globalColorPink900 = 'var(--cds-global-color-pink-900)';
export const globalColorPink1000 = 'var(--cds-global-color-pink-1000)';
export const globalColorWarmGray50 = 'var(--cds-global-color-warm-gray-50)';
export const globalColorWarmGray100 = 'var(--cds-global-color-warm-gray-100)';
export const globalColorWarmGray200 = 'var(--cds-global-color-warm-gray-200)';
export const globalColorWarmGray300 = 'var(--cds-global-color-warm-gray-300)';
export const globalColorWarmGray400 = 'var(--cds-global-color-warm-gray-400)';
export const globalColorWarmGray500 = 'var(--cds-global-color-warm-gray-500)';
export const globalColorWarmGray600 = 'var(--cds-global-color-warm-gray-600)';
export const globalColorWarmGray700 = 'var(--cds-global-color-warm-gray-700)';
export const globalColorWarmGray800 = 'var(--cds-global-color-warm-gray-800)';
export const globalColorWarmGray900 = 'var(--cds-global-color-warm-gray-900)';
export const globalColorWarmGray1000 = 'var(--cds-global-color-warm-gray-1000)';
export const globalColorSlate50 = 'var(--cds-global-color-slate-50)';
export const globalColorSlate100 = 'var(--cds-global-color-slate-100)';
export const globalColorSlate200 = 'var(--cds-global-color-slate-200)';
export const globalColorSlate300 = 'var(--cds-global-color-slate-300)';
export const globalColorSlate400 = 'var(--cds-global-color-slate-400)';
export const globalColorSlate500 = 'var(--cds-global-color-slate-500)';
export const globalColorSlate600 = 'var(--cds-global-color-slate-600)';
export const globalColorSlate700 = 'var(--cds-global-color-slate-700)';
export const globalColorSlate800 = 'var(--cds-global-color-slate-800)';
export const globalColorSlate900 = 'var(--cds-global-color-slate-900)';
export const globalColorSlate1000 = 'var(--cds-global-color-slate-1000)';
export const globalColorIce50 = 'var(--cds-global-color-ice-50)';
export const globalColorIce100 = 'var(--cds-global-color-ice-100)';
export const globalColorIce200 = 'var(--cds-global-color-ice-200)';
export const globalColorIce300 = 'var(--cds-global-color-ice-300)';
export const globalColorIce400 = 'var(--cds-global-color-ice-400)';
export const globalColorIce500 = 'var(--cds-global-color-ice-500)';
export const globalColorIce600 = 'var(--cds-global-color-ice-600)';
export const globalColorIce700 = 'var(--cds-global-color-ice-700)';
export const globalColorIce800 = 'var(--cds-global-color-ice-800)';
export const globalColorIce900 = 'var(--cds-global-color-ice-900)';
export const globalColorIce1000 = 'var(--cds-global-color-ice-1000)';
export const globalColorCoolGray50 = 'var(--cds-global-color-cool-gray-50)';
export const globalColorCoolGray100 = 'var(--cds-global-color-cool-gray-100)';
export const globalColorCoolGray200 = 'var(--cds-global-color-cool-gray-200)';
export const globalColorCoolGray300 = 'var(--cds-global-color-cool-gray-300)';
export const globalColorCoolGray400 = 'var(--cds-global-color-cool-gray-400)';
export const globalColorCoolGray500 = 'var(--cds-global-color-cool-gray-500)';
export const globalColorCoolGray600 = 'var(--cds-global-color-cool-gray-600)';
export const globalColorCoolGray700 = 'var(--cds-global-color-cool-gray-700)';
export const globalColorCoolGray800 = 'var(--cds-global-color-cool-gray-800)';
export const globalColorCoolGray900 = 'var(--cds-global-color-cool-gray-900)';
export const globalColorCoolGray1000 = 'var(--cds-global-color-cool-gray-1000)';
export const globalColorTan50 = 'var(--cds-global-color-tan-50)';
export const globalColorTan100 = 'var(--cds-global-color-tan-100)';
export const globalColorTan200 = 'var(--cds-global-color-tan-200)';
export const globalColorTan300 = 'var(--cds-global-color-tan-300)';
export const globalColorTan400 = 'var(--cds-global-color-tan-400)';
export const globalColorTan500 = 'var(--cds-global-color-tan-500)';
export const globalColorTan600 = 'var(--cds-global-color-tan-600)';
export const globalColorTan700 = 'var(--cds-global-color-tan-700)';
export const globalColorTan800 = 'var(--cds-global-color-tan-800)';
export const globalColorTan900 = 'var(--cds-global-color-tan-900)';
export const globalColorTan1000 = 'var(--cds-global-color-tan-1000)';
export const globalColorConstruction25 = 'var(--cds-global-color-construction-25)';
export const globalColorConstruction50 = 'var(--cds-global-color-construction-50)';
export const globalColorConstruction100 = 'var(--cds-global-color-construction-100)';
export const globalColorConstruction200 = 'var(--cds-global-color-construction-200)';
export const globalColorConstruction300 = 'var(--cds-global-color-construction-300)';
export const globalColorConstruction400 = 'var(--cds-global-color-construction-400)';
export const globalColorConstruction500 = 'var(--cds-global-color-construction-500)';
export const globalColorConstruction600 = 'var(--cds-global-color-construction-600)';
export const globalColorConstruction700 = 'var(--cds-global-color-construction-700)';
export const globalColorConstruction800 = 'var(--cds-global-color-construction-800)';
export const globalColorConstruction900 = 'var(--cds-global-color-construction-900)';
export const globalColorConstruction1000 = 'var(--cds-global-color-construction-1000)';
export const globalColorConstruction1100 = 'var(--cds-global-color-construction-1100)';
export const globalColorGray0 = 'var(--cds-global-color-gray-0)';
export const globalColorGray50 = 'var(--cds-global-color-gray-50)';
export const globalColorGray100 = 'var(--cds-global-color-gray-100)';
export const globalColorGray200 = 'var(--cds-global-color-gray-200)';
export const globalColorGray300 = 'var(--cds-global-color-gray-300)';
export const globalColorGray400 = 'var(--cds-global-color-gray-400)';
export const globalColorGray500 = 'var(--cds-global-color-gray-500)';
export const globalColorGray600 = 'var(--cds-global-color-gray-600)';
export const globalColorGray700 = 'var(--cds-global-color-gray-700)';
export const globalColorGray800 = 'var(--cds-global-color-gray-800)';
export const globalColorGray900 = 'var(--cds-global-color-gray-900)';
export const globalColorGray1000 = 'var(--cds-global-color-gray-1000)';
export const globalTypographyColor100 = 'var(--cds-global-typography-color-100)';
export const globalTypographyColor200 = 'var(--cds-global-typography-color-200)';
export const globalTypographyColor300 = 'var(--cds-global-typography-color-300)';
export const globalTypographyColor400 = 'var(--cds-global-typography-color-400)';
export const globalTypographyColor450 = 'var(--cds-global-typography-color-450)';
export const globalTypographyColor500 = 'var(--cds-global-typography-color-500)';
export const globalTypographyFontWeightLight = 'var(--cds-global-typography-font-weight-light)';
export const globalTypographyFontWeightRegular = 'var(--cds-global-typography-font-weight-regular)';
export const globalTypographyFontWeightMedium = 'var(--cds-global-typography-font-weight-medium)';
export const globalTypographyFontWeightSemibold = 'var(--cds-global-typography-font-weight-semibold)';
export const globalTypographyFontWeightBold = 'var(--cds-global-typography-font-weight-bold)';
export const globalTypographyFontWeightExtrabold = 'var(--cds-global-typography-font-weight-extrabold)';
export const globalTypographyFontSize0 = 'var(--cds-global-typography-font-size-0)';
export const globalTypographyFontSize1 = 'var(--cds-global-typography-font-size-1)';
export const globalTypographyFontSize2 = 'var(--cds-global-typography-font-size-2)';
export const globalTypographyFontSize3 = 'var(--cds-global-typography-font-size-3)';
export const globalTypographyFontSize4 = 'var(--cds-global-typography-font-size-4)';
export const globalTypographyFontSize5 = 'var(--cds-global-typography-font-size-5)';
export const globalTypographyFontSize6 = 'var(--cds-global-typography-font-size-6)';
export const globalTypographyFontSize7 = 'var(--cds-global-typography-font-size-7)';
export const globalTypographyFontSize8 = 'var(--cds-global-typography-font-size-8)';
export const globalTypographyFontSize9 = 'var(--cds-global-typography-font-size-9)';
export const globalTypographyFontFamily = 'var(--cds-global-typography-font-family)';
export const globalTypographyHeaderFontFamily = 'var(--cds-global-typography-header-font-family)';
export const globalTypographyMonospaceFontFamily = 'var(--cds-global-typography-monospace-font-family)';
export const globalTypographyTopGapHeight = 'var(--cds-global-typography-top-gap-height)';
export const globalTypographyAscenderHeight = 'var(--cds-global-typography-ascender-height)';
export const globalTypographyXHeight = 'var(--cds-global-typography-x-height)';
export const globalTypographyLinkColor = 'var(--cds-global-typography-link-color)';
export const globalTypographyLinkColorHover = 'var(--cds-global-typography-link-color-hover)';
export const globalTypographyLinkColorVisited = 'var(--cds-global-typography-link-color-visited)';
export const globalTypographyLinkColorVisitedHover = 'var(--cds-global-typography-link-color-visited-hover)';
export const globalTypographyLinkOnColorbg = 'var(--cds-global-typography-link-on-colorbg)';
export const globalTypographyBodyFontSize = 'var(--cds-global-typography-body-font-size)';
export const globalTypographyBodyLineHeight = 'var(--cds-global-typography-body-line-height)';
export const globalTypographyBodyLetterSpacing = 'var(--cds-global-typography-body-letter-spacing)';
export const globalTypographyBodyFontWeight = 'var(--cds-global-typography-body-font-weight)';
export const globalTypographyDisplayFontSize = 'var(--cds-global-typography-display-font-size)';
export const globalTypographyDisplayLineHeight = 'var(--cds-global-typography-display-line-height)';
export const globalTypographyDisplayLetterSpacing = 'var(--cds-global-typography-display-letter-spacing)';
export const globalTypographyDisplayFontWeight = 'var(--cds-global-typography-display-font-weight)';
export const globalTypographyHeadingFontSize = 'var(--cds-global-typography-heading-font-size)';
export const globalTypographyHeadingLineHeight = 'var(--cds-global-typography-heading-line-height)';
export const globalTypographyHeadingLetterSpacing = 'var(--cds-global-typography-heading-letter-spacing)';
export const globalTypographyHeadingFontWeight = 'var(--cds-global-typography-heading-font-weight)';
export const globalTypographyHeadlineFontSize = 'var(--cds-global-typography-headline-font-size)';
export const globalTypographyHeadlineLineHeight = 'var(--cds-global-typography-headline-line-height)';
export const globalTypographyHeadlineLetterSpacing = 'var(--cds-global-typography-headline-letter-spacing)';
export const globalTypographyHeadlineFontWeight = 'var(--cds-global-typography-headline-font-weight)';
export const globalTypographyTitleFontSize = 'var(--cds-global-typography-title-font-size)';
export const globalTypographyTitleLineHeight = 'var(--cds-global-typography-title-line-height)';
export const globalTypographyTitleLetterSpacing = 'var(--cds-global-typography-title-letter-spacing)';
export const globalTypographyTitleFontWeight = 'var(--cds-global-typography-title-font-weight)';
export const globalTypographySectionFontSize = 'var(--cds-global-typography-section-font-size)';
export const globalTypographySectionLineHeight = 'var(--cds-global-typography-section-line-height)';
export const globalTypographySectionLetterSpacing = 'var(--cds-global-typography-section-letter-spacing)';
export const globalTypographySectionFontWeight = 'var(--cds-global-typography-section-font-weight)';
export const globalTypographySubsectionFontSize = 'var(--cds-global-typography-subsection-font-size)';
export const globalTypographySubsectionLineHeight = 'var(--cds-global-typography-subsection-line-height)';
export const globalTypographySubsectionLetterSpacing = 'var(--cds-global-typography-subsection-letter-spacing)';
export const globalTypographySubsectionFontWeight = 'var(--cds-global-typography-subsection-font-weight)';
export const globalTypographyMessageFontSize = 'var(--cds-global-typography-message-font-size)';
export const globalTypographyMessageLineHeight = 'var(--cds-global-typography-message-line-height)';
export const globalTypographyMessageLetterSpacing = 'var(--cds-global-typography-message-letter-spacing)';
export const globalTypographyMessageFontWeight = 'var(--cds-global-typography-message-font-weight)';
export const globalTypographySecondaryFontSize = 'var(--cds-global-typography-secondary-font-size)';
export const globalTypographySecondaryLineHeight = 'var(--cds-global-typography-secondary-line-height)';
export const globalTypographySecondaryLetterSpacing = 'var(--cds-global-typography-secondary-letter-spacing)';
export const globalTypographySecondaryFontWeight = 'var(--cds-global-typography-secondary-font-weight)';
export const globalTypographyCaptionFontSize = 'var(--cds-global-typography-caption-font-size)';
export const globalTypographyCaptionLineHeight = 'var(--cds-global-typography-caption-line-height)';
export const globalTypographyCaptionLetterSpacing = 'var(--cds-global-typography-caption-letter-spacing)';
export const globalTypographyCaptionFontWeight = 'var(--cds-global-typography-caption-font-weight)';
export const globalTypographySmallcaptionFontSize = 'var(--cds-global-typography-smallcaption-font-size)';
export const globalTypographySmallcaptionLineHeight = 'var(--cds-global-typography-smallcaption-line-height)';
export const globalTypographySmallcaptionLetterSpacing = 'var(--cds-global-typography-smallcaption-letter-spacing)';
export const globalTypographySmallcaptionFontWeight = 'var(--cds-global-typography-smallcaption-font-weight)';
export const globalTypographyInfoHover = 'var(--cds-global-typography-info-hover)';
export const globalTypographySuccessHover = 'var(--cds-global-typography-success-hover)';
export const globalTypographyWarningHover = 'var(--cds-global-typography-warning-hover)';
export const globalTypographyDangerHover = 'var(--cds-global-typography-danger-hover)';
export const globalTypographyNeutralHover = 'var(--cds-global-typography-neutral-hover)';
export const globalTypographyDisabled = 'var(--cds-global-typography-disabled)';
export const globalTypographyDisabledButton = 'var(--cds-global-typography-disabled-button)';
export const globalAnimationDurationInstant = 'var(--cds-global-animation-duration-instant)';
export const globalAnimationDurationQuickest = 'var(--cds-global-animation-duration-quickest)';
export const globalAnimationDurationQuicker = 'var(--cds-global-animation-duration-quicker)';
export const globalAnimationDurationQuick = 'var(--cds-global-animation-duration-quick)';
export const globalAnimationDurationSecondary = 'var(--cds-global-animation-duration-secondary)';
export const globalAnimationDurationPrimary = 'var(--cds-global-animation-duration-primary)';
export const globalAnimationDurationSlow = 'var(--cds-global-animation-duration-slow)';
export const globalAnimationDurationSlower = 'var(--cds-global-animation-duration-slower)';
export const globalAnimationDurationSlowest = 'var(--cds-global-animation-duration-slowest)';
export const globalAnimationEasingPrimary = 'var(--cds-global-animation-easing-primary)';
export const globalAnimationEasingSecondary = 'var(--cds-global-animation-easing-secondary)';
export const globalAnimationEasingLoop = 'var(--cds-global-animation-easing-loop)';
export const aliasObjectBorderRadius100 = 'var(--cds-alias-object-border-radius-100)';
export const aliasObjectBorderRadius200 = 'var(--cds-alias-object-border-radius-200)';
export const aliasObjectBorderRadius300 = 'var(--cds-alias-object-border-radius-300)';
export const aliasObjectBorderWidth100 = 'var(--cds-alias-object-border-width-100)';
export const aliasObjectBorderWidth200 = 'var(--cds-alias-object-border-width-200)';
export const aliasObjectBorderWidth300 = 'var(--cds-alias-object-border-width-300)';
export const aliasObjectBorderWidth400 = 'var(--cds-alias-object-border-width-400)';
export const aliasObjectBorderColor = 'var(--cds-alias-object-border-color)';
export const aliasObjectBorderColorTint = 'var(--cds-alias-object-border-color-tint)';
export const aliasObjectBorderColorShade = 'var(--cds-alias-object-border-color-shade)';
export const aliasObjectShadow100 = 'var(--cds-alias-object-shadow-100)';
export const aliasObjectShadow200 = 'var(--cds-alias-object-shadow-200)';
export const aliasObjectShadow300 = 'var(--cds-alias-object-shadow-300)';
export const aliasObjectOpacity0 = 'var(--cds-alias-object-opacity-0)';
export const aliasObjectOpacity100 = 'var(--cds-alias-object-opacity-100)';
export const aliasObjectOpacity200 = 'var(--cds-alias-object-opacity-200)';
export const aliasObjectOpacity300 = 'var(--cds-alias-object-opacity-300)';
export const aliasObjectInteractionOutline = 'var(--cds-alias-object-interaction-outline)';
export const aliasObjectInteractionOutlineWebkit = 'var(--cds-alias-object-interaction-outline-webkit)';
export const aliasObjectInteractionOutlineOffset = 'var(--cds-alias-object-interaction-outline-offset)';
export const aliasObjectInteractionTouchTarget = 'var(--cds-alias-object-interaction-touch-target)';
export const aliasObjectInteractionBorderColor = 'var(--cds-alias-object-interaction-border-color)';
export const aliasObjectInteractionBackground = 'var(--cds-alias-object-interaction-background)';
export const aliasObjectInteractionBackgroundHover = 'var(--cds-alias-object-interaction-background-hover)';
export const aliasObjectInteractionBackgroundActive = 'var(--cds-alias-object-interaction-background-active)';
export const aliasObjectInteractionBackgroundSelected = 'var(--cds-alias-object-interaction-background-selected)';
export const aliasObjectInteractionBackgroundSelectedHover = 'var(--cds-alias-object-interaction-background-selected-hover)';
export const aliasObjectInteractionBackgroundSelectedActive = 'var(--cds-alias-object-interaction-background-selected-active)';
export const aliasObjectInteractionBackgroundDisabled = 'var(--cds-alias-object-interaction-background-disabled)';
export const aliasObjectInteractionBackgroundHighlight = 'var(--cds-alias-object-interaction-background-highlight)';
export const aliasObjectInteractionColor = 'var(--cds-alias-object-interaction-color)';
export const aliasObjectInteractionColorHover = 'var(--cds-alias-object-interaction-color-hover)';
export const aliasObjectInteractionColorActive = 'var(--cds-alias-object-interaction-color-active)';
export const aliasObjectInteractionColorSelected = 'var(--cds-alias-object-interaction-color-selected)';
export const aliasObjectInteractionColorDisabled = 'var(--cds-alias-object-interaction-color-disabled)';
export const aliasObjectInteractionInfoHover = 'var(--cds-alias-object-interaction-info-hover)';
export const aliasObjectInteractionInfoClick = 'var(--cds-alias-object-interaction-info-click)';
export const aliasObjectInteractionInfoSecondaryHover = 'var(--cds-alias-object-interaction-info-secondary-hover)';
export const aliasObjectInteractionSuccessHover = 'var(--cds-alias-object-interaction-success-hover)';
export const aliasObjectInteractionSuccessClick = 'var(--cds-alias-object-interaction-success-click)';
export const aliasObjectInteractionSuccessSecondaryHover = 'var(--cds-alias-object-interaction-success-secondary-hover)';
export const aliasObjectInteractionWarningHover = 'var(--cds-alias-object-interaction-warning-hover)';
export const aliasObjectInteractionWarningClick = 'var(--cds-alias-object-interaction-warning-click)';
export const aliasObjectInteractionWarningSecondaryHover = 'var(--cds-alias-object-interaction-warning-secondary-hover)';
export const aliasObjectInteractionDangerHover = 'var(--cds-alias-object-interaction-danger-hover)';
export const aliasObjectInteractionDangerClick = 'var(--cds-alias-object-interaction-danger-click)';
export const aliasObjectInteractionDangerSecondaryHover = 'var(--cds-alias-object-interaction-danger-secondary-hover)';
export const aliasObjectInteractionNeutralHover = 'var(--cds-alias-object-interaction-neutral-hover)';
export const aliasObjectInteractionNeutralClick = 'var(--cds-alias-object-interaction-neutral-click)';
export const aliasObjectInteractionNeutralSecondaryHover = 'var(--cds-alias-object-interaction-neutral-secondary-hover)';
export const aliasObjectInteractionInverseHover = 'var(--cds-alias-object-interaction-inverse-hover)';
export const aliasObjectInteractionInverseClick = 'var(--cds-alias-object-interaction-inverse-click)';
export const aliasObjectAppBackground = 'var(--cds-alias-object-app-background)';
export const aliasObjectOverlayBackground = 'var(--cds-alias-object-overlay-background)';
export const aliasObjectOverlayBackdropBackground = 'var(--cds-alias-object-overlay-backdrop-background)';
export const aliasObjectContainerBackground = 'var(--cds-alias-object-container-background)';
export const aliasObjectContainerBackgroundTint = 'var(--cds-alias-object-container-background-tint)';
export const aliasObjectContainerBackgroundShade = 'var(--cds-alias-object-container-background-shade)';
export const aliasObjectContainerBackgroundDark = 'var(--cds-alias-object-container-background-dark)';
export const aliasObjectContainerBackgroundInverse = 'var(--cds-alias-object-container-background-inverse)';
export const aliasObjectContainerBackgroundInverseTint = 'var(--cds-alias-object-container-background-inverse-tint)';
export const aliasObjectContainerBackgroundInverseShade = 'var(--cds-alias-object-container-background-inverse-shade)';
export const aliasObjectContainerBackdropBackground = 'var(--cds-alias-object-container-backdrop-background)';
export const aliasObjectContainerBorderColor = 'var(--cds-alias-object-container-border-color)';
export const aliasStatusInfo = 'var(--cds-alias-status-info)';
export const aliasStatusInfoTint = 'var(--cds-alias-status-info-tint)';
export const aliasStatusInfoShade = 'var(--cds-alias-status-info-shade)';
export const aliasStatusSuccess = 'var(--cds-alias-status-success)';
export const aliasStatusSuccessTint = 'var(--cds-alias-status-success-tint)';
export const aliasStatusSuccessShade = 'var(--cds-alias-status-success-shade)';
export const aliasStatusWarning = 'var(--cds-alias-status-warning)';
export const aliasStatusWarningTint = 'var(--cds-alias-status-warning-tint)';
export const aliasStatusWarningShade = 'var(--cds-alias-status-warning-shade)';
export const aliasStatusWarningDark = 'var(--cds-alias-status-warning-dark)';
export const aliasStatusDanger = 'var(--cds-alias-status-danger)';
export const aliasStatusDangerTint = 'var(--cds-alias-status-danger-tint)';
export const aliasStatusDangerShade = 'var(--cds-alias-status-danger-shade)';
export const aliasStatusDangerDark = 'var(--cds-alias-status-danger-dark)';
export const aliasStatusNeutral = 'var(--cds-alias-status-neutral)';
export const aliasStatusNeutralTint = 'var(--cds-alias-status-neutral-tint)';
export const aliasStatusNeutralShade = 'var(--cds-alias-status-neutral-shade)';
export const aliasStatusDisabled = 'var(--cds-alias-status-disabled)';
export const aliasStatusDisabledTint = 'var(--cds-alias-status-disabled-tint)';
export const aliasStatusDisabledShade = 'var(--cds-alias-status-disabled-shade)';
export const aliasStatusAlt = 'var(--cds-alias-status-alt)';
export const aliasStatusAltTint = 'var(--cds-alias-status-alt-tint)';
export const aliasStatusAltShade = 'var(--cds-alias-status-alt-shade)';
export const aliasUtilityGray = 'var(--cds-alias-utility-gray)';
export const aliasUtilityGrayTint = 'var(--cds-alias-utility-gray-tint)';
export const aliasUtilityGrayShade = 'var(--cds-alias-utility-gray-shade)';
export const aliasUtilityBlue = 'var(--cds-alias-utility-blue)';
export const aliasUtilityBlueTint = 'var(--cds-alias-utility-blue-tint)';
export const aliasUtilityBlueShade = 'var(--cds-alias-utility-blue-shade)';
export const aliasUtilityLightBlue = 'var(--cds-alias-utility-light-blue)';
export const aliasUtilityLightBlueTint = 'var(--cds-alias-utility-light-blue-tint)';
export const aliasUtilityLightBlueShade = 'var(--cds-alias-utility-light-blue-shade)';
export const aliasUtilityDarkBlue = 'var(--cds-alias-utility-dark-blue)';
export const aliasUtilityDarkBlueTint = 'var(--cds-alias-utility-dark-blue-tint)';
export const aliasUtilityDarkBlueShade = 'var(--cds-alias-utility-dark-blue-shade)';
export const aliasUtilityGreen = 'var(--cds-alias-utility-green)';
export const aliasUtilityGreenTint = 'var(--cds-alias-utility-green-tint)';
export const aliasUtilityGreenShade = 'var(--cds-alias-utility-green-shade)';
export const aliasUtilityYellow = 'var(--cds-alias-utility-yellow)';
export const aliasUtilityYellowTint = 'var(--cds-alias-utility-yellow-tint)';
export const aliasUtilityYellowShade = 'var(--cds-alias-utility-yellow-shade)';
export const aliasUtilityRed = 'var(--cds-alias-utility-red)';
export const aliasUtilityRedTint = 'var(--cds-alias-utility-red-tint)';
export const aliasUtilityRedShade = 'var(--cds-alias-utility-red-shade)';
export const aliasUtilityTangerine = 'var(--cds-alias-utility-tangerine)';
export const aliasUtilityTangerineTint = 'var(--cds-alias-utility-tangerine-tint)';
export const aliasUtilityTangerineShade = 'var(--cds-alias-utility-tangerine-shade)';
export const aliasUtilityViolet = 'var(--cds-alias-utility-violet)';
export const aliasUtilityVioletTint = 'var(--cds-alias-utility-violet-tint)';
export const aliasUtilityVioletShade = 'var(--cds-alias-utility-violet-shade)';
export const aliasVizBorder = 'var(--cds-alias-viz-border)';
export const aliasVizGeneral1Blue = 'var(--cds-alias-viz-general-1-blue)';
export const aliasVizGeneral2Violet = 'var(--cds-alias-viz-general-2-violet)';
export const aliasVizGeneral3Aqua = 'var(--cds-alias-viz-general-3-aqua)';
export const aliasVizGeneral4Lime = 'var(--cds-alias-viz-general-4-lime)';
export const aliasVizGeneral5Blue = 'var(--cds-alias-viz-general-5-blue)';
export const aliasVizGeneral6Aqua = 'var(--cds-alias-viz-general-6-aqua)';
export const aliasVizGeneral7Violet = 'var(--cds-alias-viz-general-7-violet)';
export const aliasVizGeneral8Lime = 'var(--cds-alias-viz-general-8-lime)';
export const aliasVizGeneral9Blue = 'var(--cds-alias-viz-general-9-blue)';
export const aliasVizGeneral10Lime = 'var(--cds-alias-viz-general-10-lime)';
export const aliasVizGeneral11Aqua = 'var(--cds-alias-viz-general-11-aqua)';
export const aliasVizGeneral12Violet = 'var(--cds-alias-viz-general-12-violet)';
export const aliasVizGeneral13Violet = 'var(--cds-alias-viz-general-13-violet)';
export const aliasVizGeneral14Blue = 'var(--cds-alias-viz-general-14-blue)';
export const aliasVizGeneral15Aqua = 'var(--cds-alias-viz-general-15-aqua)';
export const aliasVizGeneral16Lime = 'var(--cds-alias-viz-general-16-lime)';
export const aliasVizSeverityNeutral = 'var(--cds-alias-viz-severity-neutral)';
export const aliasVizSeveritySuccess = 'var(--cds-alias-viz-severity-success)';
export const aliasVizSeverityWarning = 'var(--cds-alias-viz-severity-warning)';
export const aliasVizSeverityImmediate = 'var(--cds-alias-viz-severity-immediate)';
export const aliasVizSeverityCritical = 'var(--cds-alias-viz-severity-critical)';
export const aliasVizSequentialBlue50 = 'var(--cds-alias-viz-sequential-blue-50)';
export const aliasVizSequentialBlue100 = 'var(--cds-alias-viz-sequential-blue-100)';
export const aliasVizSequentialBlue200 = 'var(--cds-alias-viz-sequential-blue-200)';
export const aliasVizSequentialBlue300 = 'var(--cds-alias-viz-sequential-blue-300)';
export const aliasVizSequentialBlue400 = 'var(--cds-alias-viz-sequential-blue-400)';
export const aliasVizSequentialBlue500 = 'var(--cds-alias-viz-sequential-blue-500)';
export const aliasVizSequentialBlue600 = 'var(--cds-alias-viz-sequential-blue-600)';
export const aliasVizSequentialBlue700 = 'var(--cds-alias-viz-sequential-blue-700)';
export const aliasVizSequentialBlue800 = 'var(--cds-alias-viz-sequential-blue-800)';
export const aliasVizSequentialBlue900 = 'var(--cds-alias-viz-sequential-blue-900)';
export const aliasVizSequentialBlue1000 = 'var(--cds-alias-viz-sequential-blue-1000)';
export const aliasVizSequentialViolet50 = 'var(--cds-alias-viz-sequential-violet-50)';
export const aliasVizSequentialViolet100 = 'var(--cds-alias-viz-sequential-violet-100)';
export const aliasVizSequentialViolet200 = 'var(--cds-alias-viz-sequential-violet-200)';
export const aliasVizSequentialViolet300 = 'var(--cds-alias-viz-sequential-violet-300)';
export const aliasVizSequentialViolet400 = 'var(--cds-alias-viz-sequential-violet-400)';
export const aliasVizSequentialViolet500 = 'var(--cds-alias-viz-sequential-violet-500)';
export const aliasVizSequentialViolet600 = 'var(--cds-alias-viz-sequential-violet-600)';
export const aliasVizSequentialViolet700 = 'var(--cds-alias-viz-sequential-violet-700)';
export const aliasVizSequentialViolet800 = 'var(--cds-alias-viz-sequential-violet-800)';
export const aliasVizSequentialViolet900 = 'var(--cds-alias-viz-sequential-violet-900)';
export const aliasVizSequentialViolet1000 = 'var(--cds-alias-viz-sequential-violet-1000)';
export const aliasVizSequentialOchre50 = 'var(--cds-alias-viz-sequential-ochre-50)';
export const aliasVizSequentialOchre100 = 'var(--cds-alias-viz-sequential-ochre-100)';
export const aliasVizSequentialOchre200 = 'var(--cds-alias-viz-sequential-ochre-200)';
export const aliasVizSequentialOchre300 = 'var(--cds-alias-viz-sequential-ochre-300)';
export const aliasVizSequentialOchre400 = 'var(--cds-alias-viz-sequential-ochre-400)';
export const aliasVizSequentialOchre500 = 'var(--cds-alias-viz-sequential-ochre-500)';
export const aliasVizSequentialOchre600 = 'var(--cds-alias-viz-sequential-ochre-600)';
export const aliasVizSequentialOchre700 = 'var(--cds-alias-viz-sequential-ochre-700)';
export const aliasVizSequentialOchre800 = 'var(--cds-alias-viz-sequential-ochre-800)';
export const aliasVizSequentialOchre900 = 'var(--cds-alias-viz-sequential-ochre-900)';
export const aliasVizSequentialOchre1000 = 'var(--cds-alias-viz-sequential-ochre-1000)';
export const aliasVizSequentialAqua50 = 'var(--cds-alias-viz-sequential-aqua-50)';
export const aliasVizSequentialAqua100 = 'var(--cds-alias-viz-sequential-aqua-100)';
export const aliasVizSequentialAqua200 = 'var(--cds-alias-viz-sequential-aqua-200)';
export const aliasVizSequentialAqua300 = 'var(--cds-alias-viz-sequential-aqua-300)';
export const aliasVizSequentialAqua400 = 'var(--cds-alias-viz-sequential-aqua-400)';
export const aliasVizSequentialAqua500 = 'var(--cds-alias-viz-sequential-aqua-500)';
export const aliasVizSequentialAqua600 = 'var(--cds-alias-viz-sequential-aqua-600)';
export const aliasVizSequentialAqua700 = 'var(--cds-alias-viz-sequential-aqua-700)';
export const aliasVizSequentialAqua800 = 'var(--cds-alias-viz-sequential-aqua-800)';
export const aliasVizSequentialAqua900 = 'var(--cds-alias-viz-sequential-aqua-900)';
export const aliasVizSequentialAqua1000 = 'var(--cds-alias-viz-sequential-aqua-1000)';
export const aliasVizSequentialGreen50 = 'var(--cds-alias-viz-sequential-green-50)';
export const aliasVizSequentialGreen100 = 'var(--cds-alias-viz-sequential-green-100)';
export const aliasVizSequentialGreen200 = 'var(--cds-alias-viz-sequential-green-200)';
export const aliasVizSequentialGreen300 = 'var(--cds-alias-viz-sequential-green-300)';
export const aliasVizSequentialGreen400 = 'var(--cds-alias-viz-sequential-green-400)';
export const aliasVizSequentialGreen500 = 'var(--cds-alias-viz-sequential-green-500)';
export const aliasVizSequentialGreen600 = 'var(--cds-alias-viz-sequential-green-600)';
export const aliasVizSequentialGreen700 = 'var(--cds-alias-viz-sequential-green-700)';
export const aliasVizSequentialGreen800 = 'var(--cds-alias-viz-sequential-green-800)';
export const aliasVizSequentialGreen900 = 'var(--cds-alias-viz-sequential-green-900)';
export const aliasVizSequentialGreen1000 = 'var(--cds-alias-viz-sequential-green-1000)';
export const aliasVizSequentialRed50 = 'var(--cds-alias-viz-sequential-red-50)';
export const aliasVizSequentialRed100 = 'var(--cds-alias-viz-sequential-red-100)';
export const aliasVizSequentialRed200 = 'var(--cds-alias-viz-sequential-red-200)';
export const aliasVizSequentialRed300 = 'var(--cds-alias-viz-sequential-red-300)';
export const aliasVizSequentialRed400 = 'var(--cds-alias-viz-sequential-red-400)';
export const aliasVizSequentialRed500 = 'var(--cds-alias-viz-sequential-red-500)';
export const aliasVizSequentialRed600 = 'var(--cds-alias-viz-sequential-red-600)';
export const aliasVizSequentialRed700 = 'var(--cds-alias-viz-sequential-red-700)';
export const aliasVizSequentialRed800 = 'var(--cds-alias-viz-sequential-red-800)';
export const aliasVizSequentialRed900 = 'var(--cds-alias-viz-sequential-red-900)';
export const aliasVizSequentialRed1000 = 'var(--cds-alias-viz-sequential-red-1000)';
export const aliasVizDivergingViolet-aquaV200 = 'var(--cds-alias-viz-diverging-violet-aqua-v-200)';
export const aliasVizDivergingViolet-aquaV300 = 'var(--cds-alias-viz-diverging-violet-aqua-v-300)';
export const aliasVizDivergingViolet-aquaV400 = 'var(--cds-alias-viz-diverging-violet-aqua-v-400)';
export const aliasVizDivergingViolet-aquaV500 = 'var(--cds-alias-viz-diverging-violet-aqua-v-500)';
export const aliasVizDivergingViolet-aquaV600 = 'var(--cds-alias-viz-diverging-violet-aqua-v-600)';
export const aliasVizDivergingViolet-aquaV700 = 'var(--cds-alias-viz-diverging-violet-aqua-v-700)';
export const aliasVizDivergingViolet-aquaNeutral = 'var(--cds-alias-viz-diverging-violet-aqua-neutral)';
export const aliasVizDivergingViolet-aquaA400 = 'var(--cds-alias-viz-diverging-violet-aqua-a-400)';
export const aliasVizDivergingViolet-aquaA500 = 'var(--cds-alias-viz-diverging-violet-aqua-a-500)';
export const aliasVizDivergingViolet-aquaA600 = 'var(--cds-alias-viz-diverging-violet-aqua-a-600)';
export const aliasVizDivergingViolet-aquaA700 = 'var(--cds-alias-viz-diverging-violet-aqua-a-700)';
export const aliasVizDivergingViolet-aquaA800 = 'var(--cds-alias-viz-diverging-violet-aqua-a-800)';
export const aliasVizDivergingViolet-aquaA900 = 'var(--cds-alias-viz-diverging-violet-aqua-a-900)';
export const aliasVizDivergingBlue-jadeB300 = 'var(--cds-alias-viz-diverging-blue-jade-b-300)';
export const aliasVizDivergingBlue-jadeB400 = 'var(--cds-alias-viz-diverging-blue-jade-b-400)';
export const aliasVizDivergingBlue-jadeB500 = 'var(--cds-alias-viz-diverging-blue-jade-b-500)';
export const aliasVizDivergingBlue-jadeB600 = 'var(--cds-alias-viz-diverging-blue-jade-b-600)';
export const aliasVizDivergingBlue-jadeB700 = 'var(--cds-alias-viz-diverging-blue-jade-b-700)';
export const aliasVizDivergingBlue-jadeB800 = 'var(--cds-alias-viz-diverging-blue-jade-b-800)';
export const aliasVizDivergingBlue-jadeNeutral = 'var(--cds-alias-viz-diverging-blue-jade-neutral)';
export const aliasVizDivergingBlue-jadeJ300 = 'var(--cds-alias-viz-diverging-blue-jade-j-300)';
export const aliasVizDivergingBlue-jadeJ400 = 'var(--cds-alias-viz-diverging-blue-jade-j-400)';
export const aliasVizDivergingBlue-jadeJ500 = 'var(--cds-alias-viz-diverging-blue-jade-j-500)';
export const aliasVizDivergingBlue-jadeJ600 = 'var(--cds-alias-viz-diverging-blue-jade-j-600)';
export const aliasVizDivergingBlue-jadeJ700 = 'var(--cds-alias-viz-diverging-blue-jade-j-700)';
export const aliasVizDivergingBlue-jadeJ800 = 'var(--cds-alias-viz-diverging-blue-jade-j-800)';
export const aliasVizDivergingBlue-greenB300 = 'var(--cds-alias-viz-diverging-blue-green-b-300)';
export const aliasVizDivergingBlue-greenB400 = 'var(--cds-alias-viz-diverging-blue-green-b-400)';
export const aliasVizDivergingBlue-greenB500 = 'var(--cds-alias-viz-diverging-blue-green-b-500)';
export const aliasVizDivergingBlue-greenB600 = 'var(--cds-alias-viz-diverging-blue-green-b-600)';
export const aliasVizDivergingBlue-greenB700 = 'var(--cds-alias-viz-diverging-blue-green-b-700)';
export const aliasVizDivergingBlue-greenB800 = 'var(--cds-alias-viz-diverging-blue-green-b-800)';
export const aliasVizDivergingBlue-greenNeutral = 'var(--cds-alias-viz-diverging-blue-green-neutral)';
export const aliasVizDivergingBlue-greenG300 = 'var(--cds-alias-viz-diverging-blue-green-g-300)';
export const aliasVizDivergingBlue-greenG400 = 'var(--cds-alias-viz-diverging-blue-green-g-400)';
export const aliasVizDivergingBlue-greenG500 = 'var(--cds-alias-viz-diverging-blue-green-g-500)';
export const aliasVizDivergingBlue-greenG600 = 'var(--cds-alias-viz-diverging-blue-green-g-600)';
export const aliasVizDivergingBlue-greenG700 = 'var(--cds-alias-viz-diverging-blue-green-g-700)';
export const aliasVizDivergingBlue-greenG800 = 'var(--cds-alias-viz-diverging-blue-green-g-800)';
export const aliasVizDivergingGreen-yellow-redG200 = 'var(--cds-alias-viz-diverging-green-yellow-red-g-200)';
export const aliasVizDivergingGreen-yellow-redG300 = 'var(--cds-alias-viz-diverging-green-yellow-red-g-300)';
export const aliasVizDivergingGreen-yellow-redG400 = 'var(--cds-alias-viz-diverging-green-yellow-red-g-400)';
export const aliasVizDivergingGreen-yellow-redG500 = 'var(--cds-alias-viz-diverging-green-yellow-red-g-500)';
export const aliasVizDivergingGreen-yellow-redG600 = 'var(--cds-alias-viz-diverging-green-yellow-red-g-600)';
export const aliasVizDivergingGreen-yellow-redG700 = 'var(--cds-alias-viz-diverging-green-yellow-red-g-700)';
export const aliasVizDivergingGreen-yellow-redNeutral = 'var(--cds-alias-viz-diverging-green-yellow-red-neutral)';
export const aliasVizDivergingGreen-yellow-redR400 = 'var(--cds-alias-viz-diverging-green-yellow-red-r-400)';
export const aliasVizDivergingGreen-yellow-redR500 = 'var(--cds-alias-viz-diverging-green-yellow-red-r-500)';
export const aliasVizDivergingGreen-yellow-redR600 = 'var(--cds-alias-viz-diverging-green-yellow-red-r-600)';
export const aliasVizDivergingGreen-yellow-redR700 = 'var(--cds-alias-viz-diverging-green-yellow-red-r-700)';
export const aliasVizDivergingGreen-yellow-redR800 = 'var(--cds-alias-viz-diverging-green-yellow-red-r-800)';
export const aliasVizDivergingGreen-yellow-redR900 = 'var(--cds-alias-viz-diverging-green-yellow-red-r-900)';
export const aliasTypographyColor100 = 'var(--cds-alias-typography-color-100)';
export const aliasTypographyColor200 = 'var(--cds-alias-typography-color-200)';
export const aliasTypographyColor300 = 'var(--cds-alias-typography-color-300)';
export const aliasTypographyColor400 = 'var(--cds-alias-typography-color-400)';
export const aliasTypographyColor450 = 'var(--cds-alias-typography-color-450)';
export const aliasTypographyColor500 = 'var(--cds-alias-typography-color-500)';
export const aliasTypographyFontWeightLight = 'var(--cds-alias-typography-font-weight-light)';
export const aliasTypographyFontWeightRegular = 'var(--cds-alias-typography-font-weight-regular)';
export const aliasTypographyFontWeightMedium = 'var(--cds-alias-typography-font-weight-medium)';
export const aliasTypographyFontWeightSemibold = 'var(--cds-alias-typography-font-weight-semibold)';
export const aliasTypographyFontWeightBold = 'var(--cds-alias-typography-font-weight-bold)';
export const aliasTypographyFontWeightExtrabold = 'var(--cds-alias-typography-font-weight-extrabold)';
export const aliasTypographyFontSize0 = 'var(--cds-alias-typography-font-size-0)';
export const aliasTypographyFontSize1 = 'var(--cds-alias-typography-font-size-1)';
export const aliasTypographyFontSize2 = 'var(--cds-alias-typography-font-size-2)';
export const aliasTypographyFontSize3 = 'var(--cds-alias-typography-font-size-3)';
export const aliasTypographyFontSize4 = 'var(--cds-alias-typography-font-size-4)';
export const aliasTypographyFontSize5 = 'var(--cds-alias-typography-font-size-5)';
export const aliasTypographyFontSize6 = 'var(--cds-alias-typography-font-size-6)';
export const aliasTypographyFontSize7 = 'var(--cds-alias-typography-font-size-7)';
export const aliasTypographyFontSize8 = 'var(--cds-alias-typography-font-size-8)';
export const aliasTypographyFontSize9 = 'var(--cds-alias-typography-font-size-9)';
export const aliasTypographyFontFamily = 'var(--cds-alias-typography-font-family)';
export const aliasTypographyHeaderFontFamily = 'var(--cds-alias-typography-header-font-family)';
export const aliasTypographyMonospaceFontFamily = 'var(--cds-alias-typography-monospace-font-family)';
export const aliasTypographyTopGapHeight = 'var(--cds-alias-typography-top-gap-height)';
export const aliasTypographyAscenderHeight = 'var(--cds-alias-typography-ascender-height)';
export const aliasTypographyXHeight = 'var(--cds-alias-typography-x-height)';
export const aliasTypographyLinkColor = 'var(--cds-alias-typography-link-color)';
export const aliasTypographyLinkColorHover = 'var(--cds-alias-typography-link-color-hover)';
export const aliasTypographyLinkColorVisited = 'var(--cds-alias-typography-link-color-visited)';
export const aliasTypographyLinkColorVisitedHover = 'var(--cds-alias-typography-link-color-visited-hover)';
export const aliasTypographyLinkOnColorbg = 'var(--cds-alias-typography-link-on-colorbg)';
export const aliasTypographyBodyFontSize = 'var(--cds-alias-typography-body-font-size)';
export const aliasTypographyBodyLineHeight = 'var(--cds-alias-typography-body-line-height)';
export const aliasTypographyBodyLetterSpacing = 'var(--cds-alias-typography-body-letter-spacing)';
export const aliasTypographyBodyFontWeight = 'var(--cds-alias-typography-body-font-weight)';
export const aliasTypographyDisplayFontSize = 'var(--cds-alias-typography-display-font-size)';
export const aliasTypographyDisplayLineHeight = 'var(--cds-alias-typography-display-line-height)';
export const aliasTypographyDisplayLetterSpacing = 'var(--cds-alias-typography-display-letter-spacing)';
export const aliasTypographyDisplayFontWeight = 'var(--cds-alias-typography-display-font-weight)';
export const aliasTypographyHeadingFontSize = 'var(--cds-alias-typography-heading-font-size)';
export const aliasTypographyHeadingLineHeight = 'var(--cds-alias-typography-heading-line-height)';
export const aliasTypographyHeadingLetterSpacing = 'var(--cds-alias-typography-heading-letter-spacing)';
export const aliasTypographyHeadingFontWeight = 'var(--cds-alias-typography-heading-font-weight)';
export const aliasTypographyHeadlineFontSize = 'var(--cds-alias-typography-headline-font-size)';
export const aliasTypographyHeadlineLineHeight = 'var(--cds-alias-typography-headline-line-height)';
export const aliasTypographyHeadlineLetterSpacing = 'var(--cds-alias-typography-headline-letter-spacing)';
export const aliasTypographyHeadlineFontWeight = 'var(--cds-alias-typography-headline-font-weight)';
export const aliasTypographyTitleFontSize = 'var(--cds-alias-typography-title-font-size)';
export const aliasTypographyTitleLineHeight = 'var(--cds-alias-typography-title-line-height)';
export const aliasTypographyTitleLetterSpacing = 'var(--cds-alias-typography-title-letter-spacing)';
export const aliasTypographyTitleFontWeight = 'var(--cds-alias-typography-title-font-weight)';
export const aliasTypographySectionFontSize = 'var(--cds-alias-typography-section-font-size)';
export const aliasTypographySectionLineHeight = 'var(--cds-alias-typography-section-line-height)';
export const aliasTypographySectionLetterSpacing = 'var(--cds-alias-typography-section-letter-spacing)';
export const aliasTypographySectionFontWeight = 'var(--cds-alias-typography-section-font-weight)';
export const aliasTypographySubsectionFontSize = 'var(--cds-alias-typography-subsection-font-size)';
export const aliasTypographySubsectionLineHeight = 'var(--cds-alias-typography-subsection-line-height)';
export const aliasTypographySubsectionLetterSpacing = 'var(--cds-alias-typography-subsection-letter-spacing)';
export const aliasTypographySubsectionFontWeight = 'var(--cds-alias-typography-subsection-font-weight)';
export const aliasTypographyMessageFontSize = 'var(--cds-alias-typography-message-font-size)';
export const aliasTypographyMessageLineHeight = 'var(--cds-alias-typography-message-line-height)';
export const aliasTypographyMessageLetterSpacing = 'var(--cds-alias-typography-message-letter-spacing)';
export const aliasTypographyMessageFontWeight = 'var(--cds-alias-typography-message-font-weight)';
export const aliasTypographySecondaryFontSize = 'var(--cds-alias-typography-secondary-font-size)';
export const aliasTypographySecondaryLineHeight = 'var(--cds-alias-typography-secondary-line-height)';
export const aliasTypographySecondaryLetterSpacing = 'var(--cds-alias-typography-secondary-letter-spacing)';
export const aliasTypographySecondaryFontWeight = 'var(--cds-alias-typography-secondary-font-weight)';
export const aliasTypographyCaptionFontSize = 'var(--cds-alias-typography-caption-font-size)';
export const aliasTypographyCaptionLineHeight = 'var(--cds-alias-typography-caption-line-height)';
export const aliasTypographyCaptionLetterSpacing = 'var(--cds-alias-typography-caption-letter-spacing)';
export const aliasTypographyCaptionFontWeight = 'var(--cds-alias-typography-caption-font-weight)';
export const aliasTypographySmallcaptionFontSize = 'var(--cds-alias-typography-smallcaption-font-size)';
export const aliasTypographySmallcaptionLineHeight = 'var(--cds-alias-typography-smallcaption-line-height)';
export const aliasTypographySmallcaptionLetterSpacing = 'var(--cds-alias-typography-smallcaption-letter-spacing)';
export const aliasTypographySmallcaptionFontWeight = 'var(--cds-alias-typography-smallcaption-font-weight)';
export const aliasTypographyInfoHover = 'var(--cds-alias-typography-info-hover)';
export const aliasTypographySuccessHover = 'var(--cds-alias-typography-success-hover)';
export const aliasTypographyWarningHover = 'var(--cds-alias-typography-warning-hover)';
export const aliasTypographyDangerHover = 'var(--cds-alias-typography-danger-hover)';
export const aliasTypographyNeutralHover = 'var(--cds-alias-typography-neutral-hover)';
export const aliasTypographyDisabled = 'var(--cds-alias-typography-disabled)';
export const aliasTypographyDisabledButton = 'var(--cds-alias-typography-disabled-button)';
export const internalScale1 = 'var(--cds-internal-scale-1)';
export const internalScale2 = 'var(--cds-internal-scale-2)';
export const internalScale3 = 'var(--cds-internal-scale-3)';
